packages/web/src/components/edit/fields/visibility/VisibilityField.tsx

```diff
@@ -25,27 +26,15 @@ import { formatCalendarTime } from 'utils/dateUtils'
 
 import { IS_PRIVATE, IS_SCHEDULED_RELEASE, IS_UNLISTED } from '../types'
 
-import { ReleaseDateField } from './ReleaseDateField'
+import { ScheduledReleaseDateField } from './ScheduledReleaseDateField'
 import { mergeReleaseDateValues } from './mergeReleaseDateValues'
 
 const { requestOpen: openEditAccessConfirmationModal } =
   editAccessConfirmationModalUIActions
 
 const messages = {
-  title: 'Visibility',
-  description:
-    'Change the visibility of this release or schedule it to release in the future.',
-  public: 'Public',
-  publicDescription: 'Visible to everyone on Audius.',
-  hidden: 'Hidden',
-  hiddenDescription:
-    'Only you and people you share a direct link with will be able to listen.',
-  scheduled: (date: string) => `Scheduled for ${formatCalendarTime(date)}`,
-  scheduledRelease: 'Scheduled Release',
-  scheduledReleaseDescription:
-    'Select the date and time this will become public.',
-  hiddenHint: (entityType: 'track' | 'album' | 'playlist') =>
-    `You can’t make a public ${entityType} hidden`
+  ...visibilityMessages,
+  scheduled: (date: string) => `Scheduled for ${formatCalendarTime(date)}`
```

Comment:
is there a reason this doesn't belong in `visibillityMessages`?

Reply:
formatCalendarTime uses moment and i didnt want to introduce that into common. additionally mobile was doing something different so this is what i went with. but yes ideally this should be in common.